src/thing_creature.c

```diff
@@ -6133,34 +6133,150 @@ void process_magic_fall_effect(struct Thing *thing)
     }
 }
 
+TbBool cube_castability_can_target_creature(struct Thing *thing, PlayerNumber plyr_idx, unsigned short castability_flags)
+{
```

Comment:
You could probably return false right away if the castability flag equals 'none'.